package.json

```diff
@@ -32,24 +32,26 @@
     },
     "homepage": "https://github.com/mertkahyaoglu/storybook-zeplin#readme",
     "peerDependencies": {
-        "@storybook/addons": "^6.1.10",
-        "@storybook/api": "^6.1.10",
-        "@storybook/components": "^6.1.10",
-        "@storybook/theming": "^6.1.10",
-        "react": ">=16.8.0"
+        "@storybook/addons": "^6.0.0",
+        "@storybook/api": "^6.0.0",
+        "@storybook/components": "^6.0.0",
+        "@storybook/theming": "^6.0.0",
+        "react": ">=16.8.0",
+        "react-dom": ">=16.8.0"
```

Comment:
react-dom was not present as a peer dependency, meaning that users who didn't have it would have their project break since the newly release feature for image overlays (that requires react-dom). I'm not 100% sure about the version requirement though, I just used the same as `react`.